pkg/neonvm/controllers/vm_controller.go

```diff
@@ -1474,10 +1382,9 @@ func podSpec(
 					Command: func() []string {
 						cmd := []string{"runner"}
 						if config.DisableRunnerCgroup {
-							cmd = append(cmd, "-skip-cgroup-management")
 							// cgroup management disabled, but we still need something to provide
 							// the server, so the runner will just provide a dummy implementation.
```

Comment:
question: Is this comment still valid now that `-enable-dummy-cpu-server` was removed?

Reply:
droped comment